app/main.py

```diff
@@ -1,13 +0,0 @@
-# from models import Actor
```

Comment:
you didn't have to delete this file - just needed to revert changes that you did. Make it look like the file on master and that's it

Reply:
revert deletion of this file and i'll aprove